ckanext/datapub/templates/external_storage/snippets/upload_module.html

```diff
@@ -5,10 +5,10 @@
 {% resource 'datapub/js/main.c4752953.chunk.js' %}
 
 <div id="ResourceEditor"
-     data-dataset-id="{{ pkg_name }}"
+     data-dataset-id="{{ pkg_id }}"
      data-api="{{ base_url }}"
-     data-lfs="{{ h.extstorage_lfs_url() }}"
+     data-lfs="{{ h.extstorage_server_url() }}"
      data-auth-token="{{ api_key }}"
-     data-organization-id="{{ h.extstorage_organization_name(pkg_name) }}"
+     data-organization-id="{{ h.extstorage_storage_namespace() }}"
```

Comment:
@shevron I found the `extstorage_server_url` helper in [ckanext-blob-storage](https://github.com/datopian/ckanext-blob-storage/blob/master/ckanext/external_storage/plugin.py#L31) but I didn't find `extstorage_storage_namespace` where is it coming from?

Reply:
This is referring to a new branch of ckanext-blob-storage, which has not been merged yet. I want to merge both PRs together.

Reply:
nice, for me looks good.